demos/select.scss

```diff
@@ -29,5 +29,11 @@
   // Focused colors
   @include mdc-select-focused-bottom-line-color(green);
   @include mdc-select-focused-label-color(green);
+
+  // Menu colors
+  @include mdc-select-menu-item-color(blue);
```

Comment:
Instead of calling new mixins, and creating new mixins....just call the existing mdc-list mixins with the `mdc-select__menu` selector

```
.mdc-select__menu .mdc-list-item {
  @include mdc-list-item-primary-text-ink-color(blue);
}
```

...if a mixin in mdc-list does not exist, then we should wait for more customization in mdc-list. So don't add new mixins to mdc-select